projects/plugins/boost/changelog/add-show-performance-improvement

```diff
@@ -0,0 +1,4 @@
+Significance: patch
```

Comment:
I think this one can be `minor`, since we're going to release it as 1.3.0 when it's ready 😁

Reply:
I will update before finishing. 😄🙂